tests/modules/test_cloudProvisioning.py

```diff
@@ -828,10 +830,10 @@ def test_cloud_paths(self):
                 # all resources on the disk, dir has been packed
                 path_to_file = get_full_path(self.obj.engine.find_file(old_name))
                 msg = 'File %s (%s) not found on disk' % (old_name, path_to_file)
-                self.assertTrue(os.path.exists(path_to_file), msg)
+                self.assertTrue(os.path.exists(path_to_file) or jmeter_var_pattern.search(path_to_file), msg)
                 msg = 'Short name %s not found in modified config' % new_name
                 self.assertIn(new_name, str_cfg, msg)  # all short names in config
-                if new_name != old_name:
+                if new_name != old_name and not jmeter_var_pattern.search(new_name):
```

Comment:
If jmeter_var_pattern is found in the path old and new names can't be different (we may not change path), so check for pattern must be removed from test and MasterProvisioning._fix_filenames() need to be fixed.
It illustrates some limitation of usage of variables in the paths for cloud prov: only files (not dirs!), located in the dir of jmx file can be pointed with JMeter variables (in local prov we can allow absolute paths additionally, relative paths are allowed only for exist jmxs).